moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/javamodel/Helper.java

```diff
@@ -317,8 +318,9 @@ public boolean isAnnotationPresent(JavaHasAnnotations element, Class annotationC
      *     JavaClass' raw name
      * 2 - the provided JavaClass' raw name starts with "java."
      * 3 - the provided JavaClass' raw name starts with "javax.", with
-     *     the exception of "javax.xml.ws."
-     *
+     *     the exception of "javax.xml.ws." and "javax.xml.rpc"
+     * Note : jax-rpc should not be considered as built-in type. The API itself is 
```

Comment:
remove this note as ALL java/javax packages NOT available in the JDK should be considered as not-builtin